### PR TITLE
Fixes missing log line for deleteUser

### PR DESCRIPTION
**Pull request by dieulo**
This PR fixes an issue where Server Log was not reporting **deleteUSER** request

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ func main() {
 	router.HandleFunc("/users", createUser).Methods("POST")
 	router.HandleFunc("/users/{id}", updateUser).Methods("PUT")
 	router.HandleFunc("/users/{id}", deleteUser).Methods("DELETE")
-
+	// nerd
```
Comment (digivava) [inline main.go]: You probably meant not to include this, can you remove it?

**Review by digivava — CHANGES_REQUESTED** [review]
This looks great! Just one accidental comment to remove, then I'll approve it after you fix that.

File: main.go (hunk 1 above)
Comment (dieulo) [reply main.go]: Thank you for noticing! It was a side request from the team, but I can remove it if needed.

**Review by digivava — APPROVED** [review]
This is much better, thank you!

**Pull request merged**